superset/migrations/shared/utils.py

```diff
@@ -406,3 +406,33 @@ def drop_index(table_name: str, index_name: str) -> None:
     )
 
     op.drop_index(table_name=table_name, index_name=index_name)
+
+
+def drop_fks_for_table_by_names(table_name: str, constraint_names: list[str]) -> None:
```

Comment:
I think it would be better in terms of simplicity and to reduce lines of code to modify the original drop_fks_for_table to accept an optional list of foreign key names. You can make it clear in the docs string that if a list is not supplied, all foreign keys will be deleted.

Reply:
Sounds good, will also have to add one for creating the keys since `sqllite` doesn't like constraint AT ALL :(

Reply:
Ok @michael-s-molina the PR was updated to use the utils now. Thanks